tools/src/main/java/org/apache/kafka/trogdor/workload/ConnectionStressWorker.java

```diff
@@ -185,6 +194,15 @@ private boolean attemptConnection(AdminClientConfig conf,
                 return false;
             }
         }
+
+        private boolean attemptMetadataFetch(Properties conf) throws Exception {
+            try (AdminClient client = AdminClient.create(conf)){
+                client.describeCluster().nodes().get();
+            } catch (Exception e) {
```

Comment:
spotbugs: doesn't like `Exception` since it is not thrown in the block, change to `RuntimeException`?